### ISSUE TITLE
[Bug]: [REGRESSION] Unable to configure Elasticsearch datasource

### ISSUE THREAD

**Issue by rohan-arthur** [appsmithorg/appsmith#13919]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

When I try to add a new Elasticsearch datasource, I am not able to enter anything in the host url and port number. Here's the video with commentary:
[![LOOM DEMO](http://cdn.loom.com/sessions/thumbnails/67327345256f4109b513b0c21c5f6902-00001.gif)](https://www.loom.com/share/67327345256f4109b513b0c21c5f6902)

Tried a couple of other plugins and so far this seems to be affecting only elasticsearch

### Steps To Reproduce

1. open your app
2. add new datasource
3. select `elasticsearch`
4. try to enter host url or port number

Entry is blocked, the input fields don't reach to keystrokes

### Version

Cloud

**Comment by prapullac** [comment]
We are unable to enter the keystroke in the Host fields hence reopening the issue. Attaching video below :
https://www.loom.com/share/5789e6660e094b448e81f7826ba32eee

**Comment by prapullac** [comment]
It is working on DP and waiting on it to merge into release hence closing the issue
